Framework/Crystal/src/PredictPeaks.cpp

```diff
@@ -627,6 +627,7 @@ void PredictPeaks::calculateQAndAddToOutput(const V3D &hkl,
   // Save the run number found before.
   peak->setRunNumber(m_runNumber);
   peak->setHKL(hkl * m_qConventionFactor);
+  peak->setIntHKL(hkl * m_qConventionFactor);
```

Comment:
Would it make sense for `Peaks::setHKL` to call `Peaks::setIntHKL` internally or vice versa? Or is there a case where one would want them to different?

Reply:
@quantumsteve Some of the modulation vectors are 0.5 and above, so we cannot round the HKL and always get the correct int HKL.

Reply:
:+1: